packages/devtools/client/src/components/Devtools/Capsule.module.scss

```diff
@@ -26,22 +26,21 @@
   border-radius: 999px;
   cursor: pointer;
   touch-action: none;
-}
-
-.fab :global(*) {
+  padding: 0 var(--space-1);
+  transition: box-shadow 400ms;
   user-select: none;
-  pointer-events: none;
+  &:hover {
```

Comment:
should add empty line between css selectors

Reply:
done